src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
It seems like you are using BEM methodology for naming your classes. That's good! But, according to BEM, modifiers should be key-value pairs. So, instead of 'calendar--start-day-sun' it should be something like 'calendar--start-day_sun'. The same applies to 'calendar--month-length-31'.